docs/sources/tutorials/send-metrics-to-prometheus.md

```diff
@@ -0,0 +1,182 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tutorials/send-metrics-to-prometheus/
+description: Learn how to send metrics to Prometheus
+menuTitle: Send metrics to Prometheus
+title: Send 
```

Comment:
```suggestion
title: Send metrics to Prometheus
```

Reply:
Yes absolutely we can change the title to be more descriptive. Make sure that the Heading 1 matches the `title:` metadata. If the descriptive title gets too long we can add a `menuTitle:` to the metadata for a shorter TOC menu item.  For example:

```
title: Use Grafana Alloy to send metrics to Prometheus
menuTitle: Send metrics to Prometheus
```